### DISCUSSION TITLE
已付款多日未到账

### DISCUSSION THREAD

**Discussion by zannenkiki** [getlantern/lantern-forum-cn#870] [Q&A]
![微信图片_20240916191018](https://github.com/user-attachments/assets/ab8fcbd6-140e-4ffe-8d53-1d79b5277292)
请帮忙处理下 @Mike777Khoo

**Comment by MaxMKMax** [comment]
@zannenkiki 您好. 请您也提供我账号邮箱. 我帮您提交问题上去.

**Reply by zannenkiki** [reply]
zq826@126.com

**Reply by MaxMKMax** [reply]
@zannenkiki 好的谢谢! 马上帮您提交上去.

**Reply by zannenkiki** [reply]
专业版依然没到账，请帮忙

**Reply by zannenkiki** [reply]
@Mike777Khoo 请问何时能帮我续上专业版呢

**Reply by zannenkiki** [reply]
已成功续费，谢谢

**Reply by MaxMKMax** [reply]
@zannenkiki 好的. 不客气呢. 如果还有什么问题请来找我们. 谢谢.

**Comment by allenwork0001** [comment]
不是400多2年吗，已经没有优惠了吗？

**Reply by MaxMKMax** [reply]
@allenwork0001 您方便说是哪个优惠吗? 我这里不是很清楚. 但我可以帮您问看.
